actors/builtin/miner/miner_test.go

```diff
@@ -774,9 +774,54 @@ func TestWindowPost(t *testing.T) {
 
 		// Verify proof recorded
 		deadline := actor.getDeadline(rt, dlIdx)
-		empty, err := deadline.PostSubmissions.IsEmpty()
+		assertBitfieldEquals(t, deadline.PostSubmissions, pIdx)
+
+		// Advance to end-of-deadline cron to verify no penalties.
+		advanceDeadline(rt, actor, &cronConfig{})
+	})
+
+	t.Run("test duplicate proof ignored", func(t *testing.T) {
+		rt := builder.Build(t)
+		actor.constructAndVerify(rt)
+		store := rt.AdtStore()
+		sector := actor.commitAndProveSectors(rt, 1, 181, nil)[0]
+
+		st := getState(rt)
+		dlIdx, pIdx, err := st.FindSector(store, sector.SectorNumber)
 		require.NoError(t, err)
-		assert.False(t, empty, "no post submission")
+
+		// Skip over deadlines until the beginning of the one with the new sector
```

Comment:
At this point, an `advanceToDeadline(rt, actor, dlIdx)` that assumes empty cron configs would be handy.